### PR TITLE
Telegram bots 7.X.X

### PR DESCRIPTION
**Pull request by rubenlagus**
New redesign of Telegram Bots

### CODE REVIEW

File: telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java [hunk 1]
```diff
@@ -11,14 +11,31 @@
...
         super(options);
     }
 
+    public TelegramLongPollingBot(String botToken) {
+        this(new DefaultBotOptions(), botToken);
+    }
+    public TelegramLongPollingBot(DefaultBotOptions options, String botToken) {
+        super(options, botToken);
+    }
+
```
Comment (timursergeevichname) [inline telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java:38]: need to add translation after line 34

Comment (rubenlagus) [reply telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java:38]: What do you mean?

Comment (timursergeevichname) [reply telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java:38]: [HERE](https://ibb.co/FKD3pB2)

Comment (Chase22) [reply telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java:38]: What you you mean by translations

Comment (timursergeevichname) [reply telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java:38]: \n character

Comment (Chase22) [reply telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java:38]: Oh a line break. Yeah there's an empty line missing between the methods

Comment (timursergeevichname) [reply telegrambots/src/main/java/org/telegram/telegrambots/bots/TelegramLongPollingBot.java:38]: Thanks. I will remember :)

**Pull request closed without merging**